doc/man/zcash-cli.1

```diff
@@ -3,7 +3,7 @@
 .SH NAME
 zcash-cli \- manual page for zcash-cli v1.0.7
 .SH DESCRIPTION
-Zcash RPC client version v1.0.7
+Zcash RPC client version v1.0.7\-24a24abf7\-dirty
```

Comment:
That makes sense to me, could we generate them during the local build process for zcash instead of in the release? We'd have to generate them for the debian package as well in that case